docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+Now, if you need to add images to the file, there is a folder called `assets`, the `Path: ./3.0.0-beta.x/assets`. When you add images, these images need are added to a new folder in the `assets` folder. The name of the folder, should match the name of the file and be placed in a matching directory to the directory of the file.
+
+For example, the `example-tutorial.md` file is located in the `/getting-started/` directory.
+
+```js
+module.exports = [
+  {
+    collapsable: false,
+    title: '🚀 Getting started',
+    children: [
+      '/3.0.0-beta.x/getting-started/example-tutorial',  // <- Added here
+    ],
+  },
+
+```
+
+Therefore, images are placed in `./assets/getting-started/example-tuturial/image.png`, like this:
+
+```
+DOCS
+└───.vuepress
+└───.3.0.0-beta.x
+│   └───getting-started
+│   │   └───example-tutorial.md         // Markdown File
+│   │
+│   └───assets                          // Image Assets
+│   │   └───getting-started             // Directory of the Markdown file
+│   │       └───example-tutorial        // Name of the Markdown File
+│   │           image1.jpg
+│   │           image2.jpg
+│   │           image2.jpg
+│   │
+│   └───etc
+```
+
+**Note:** Images are not linked outside the documentation, but **can** link to an image saved in a different directory. Mainly if the image already exists.
+
+These above guidelines illustrate how to add pages, images, and the structure of the documentation. The documentation is in a language called [Markdown](https://guides.github.com/features/mastering-markdown/), you can do this [Markdown Tutorial](https://www.markdowntutorial.com) to learn how to use Markdown.
+
+### Submitting new documentation
+
+Submitting new documentation generally involves submitting new [Tutorials](/3.0.0-beta.x/community/contribute-with-docs.html#tutorials) or new [How-tos](/3.0.0-beta.x/community/contribute-with-docs.html#how-to-s).
+
+**NOTE:** If you are providing _new examples_ for existing [Reference Topics](http://localhost:8080/documentation/3.0.0-beta.x/community/contribute-with-docs.html#reference-topics), then the next section on [adding, modifying and deleting content](http://localhost:8080/documentation/3.0.0-beta.x/community/contribute-with-docs.html#adding-modifying-deleting-content-from-the-docs) applies.
+
+- Before you start to determine if you're writing a new **Tutorial** or a new **How-to**
+- Outline the **Tutorial** or a new **How-to**.
+- Speak to a Strapi Maintainer or open an issue on GitHub with the outline to receive feedback.
+- Create a GitHub repo with your app for the **Tutorial** or for the code for the **How-to**.
+- Show this code to the Strapi Maintainer and address any feedback
+- Following the [Docs Style Guide](http://localhost:8080/documentation/3.0.0-beta.x/community/contribute-with-docs.html#docs-style-guide) below, write your Tutorial and make a pull request.
+
+#### Tips
```

Comment:
I think you should sum up the content of this section

Reply:
I added the following summary. 
```
The above steps allow proceeding confidently, knowing your contribution can be accepted.
```